tracing-core/src/callsite.rs

```diff
@@ -72,7 +72,7 @@ impl Registry {
 
 /// Trait implemented by callsites.
 ///
-/// These functions are only intended to be called by the [`Registry`] which
+/// These functions are only intended to be called by the `Registry` which
```

Comment:
I think these docs are outdated, since there isn't a public registry type exposed by this crate (and hasn't been since it was initially released), let's change this to 
```suggestion
/// These functions are only intended to be called by the callsite registry, which
```
or something?